### ISSUE TITLE
[raxda] Create 3-4 second left/right image data bag

### ISSUE THREAD

**Issue by l0g1x** [chicagoedt/revo_robot#11]
Basheer and Joel, Can you take care of this for me? I need to look at the cpu usage on the radxa board for the stereo_image_proc node. Joel, Basheer can show you how to bag ("log") ros data.

**Comment by bsubei** [comment]
a 34 second bag has been created (sorry if it's too long). I also learned that rosbag recording ALL topics results in choppy framerate for the images. When you only record the specific topics (image_raw left and right, and disparity in our case) the framerate is smooth.

**Comment by oxoocoffee** [comment]
I just found out in October there is new Radxa coming out. 4G RAM, OpenCL 1.1, A17 CPU, 16 MMC internal slash, 1G Ethernet, SATA Port

http://radxa.com/Rock2

-- 
Robert J. Gebis
Sent with Airmail

On September 23, 2014 at 15:57:31, basheersubei (notifications@github.com) wrote:

a 34 second bag has been created (sorry if it's too long). I also learned that rosbag recording ALL topics results in choppy framerate for the images. When you only record the specific topics (image_raw left and right, and disparity in our case) the framerate is smooth.
